docs/content/0.1.0/docs/getting-started/creating-nodes/cubes.md

```diff
@@ -42,10 +42,14 @@ curl -X POST http://localhost:8000/nodes/cube/ \
 {{< tab "python" >}}
 
 ```py
-dj.new_cube(
+from datajunction import DJBuilder, NodeMode
+dj = DJBuilder(DJ_URL)
+
+repairs_cube = dj.create_cube(
     name="repairs_cube",
     display_name="Repairs Cube",
     description="Cube of various metrics related to repairs",
+    mode=NodeMode.PUBLISHED,
```

Comment:
Since PUBLISHED is the default do we want to maybe show the non-default way?

Reply:
Good point! I updated this to add a comment on the mode, indicating that it can be set to `NodeMode.DRAFT`.